docs/chapter2-payload-image-format.rst

```diff
@@ -0,0 +1,662 @@
+                                              must be provided for the 'firmware' image in
+                                              a configuration, but is optional for others.
+                                              Images without a load address can be loaded
+                                              to any suitable location. They can also be
+                                              left where they are in the FIT, i.e. not
+                                              loaded at all.
+   project           R           string       Type of the image, indicating which project
+                                              produced it:
+                                              "tianocore" - UEFI binary
+                                              "u-boot" - U-Boot
+                                              "op-tee" - Open Trusted Execution
+                                              Environment
+                                              "opensbi" - RISC-V OpenSBI
+                                              "arm-trusted-firmware" - ARM Trusted
+                                              Firmware
+                                              “linuxboot” - Linuxboot
+   capabilities      O           string list  List of capabilities that the Payload has:
+                                              (not defined)
+   producer          O           string       Indicates the build system and version
+                                              which produced the FIT
+   uncomp-size       R / O       u32          Size of the uncompressed data in bytes. If
+                                              the data is not compressed, this can be
+                                              omitted.
+   entry-start       O           u32 / u64    If required, this is the offset of the image
+                                              entry point from the load address of the
+                                              image. For example, a value of 0x10 means
+                                              that the image entry point is 16 bytes after
+                                              the start of the image. If omitted, a value
+                                              of 0 is assumed.
+   reloc-start       O           u32 / u64    If the image supports relocation, this is
+                                              the offset of the start of the relocation
+                                              data within the image.
+                                              See "Relocation" for information about how
+                                              to use this.
+   ================= =========== ============ ==============================================
+
+.. note:: 
+
+   #. Relocation is described below, in the section called "Relocation". The
+      "binary-self-reloc" type can be loaded to any word-aligned address. The
```

Comment:
"binary-self-reloc" or "binary" are not in the spec, instead, now we should base on "load" property to determine if image can be loaded anyway or must be loaded to specific address.

Reply:
I moved this into the table. Is it better?

Reply:
which table?

Reply:
OK. found it. Looks good! Thanks!